custom_platforms.js

```diff
@@ -223,7 +235,9 @@ module.exports = {
                 "\\$protobuf",
                 "important,protobuf",
                 "\\$extension",
-                ",extension"
+                ",extension",
+                "\\$(.*,)?hls=",
```

Comment:
why not more simple few separate patterns? just like for other modifiers
```
"\\$hls="
",hls=",
```

Reply:
copy-pasta of existing modifiers

Reply:
changed